components/core/src/clp/clo/CommandLineArguments.cpp

```diff
@@ -350,8 +387,7 @@ CommandLineArguments::parse_arguments(int argc, char const* argv[]) {
             throw invalid_argument(
                     "Aggregations are only supported with the reducer output handler."
             );
-        } else if ((false == m_do_count_results_aggregation
-                    && OutputHandlerType::Reducer == m_output_handler_type))
+        } else if (false == m_do_count_results_aggregation && OutputHandlerType::Reducer == m_output_handler_type)
```

Comment:
Let's undo this (we need the extra brackets so that `clang-format` formats this correctly).